jobs/e2e_node/containerd/config.toml

```diff
@@ -1,5 +1,3 @@
-version = 2
```

Comment:
why do we need to revert to v1?

Reply:
v1 is the version currently used for containerd-related jobs. I reverted so we don't have too much impact to the overall test-infra. I can work on a upgrade to the v2 if that's what we want.